src/components/profile/PreviewAs.vue

```diff
@@ -79,15 +72,36 @@ export default {
     },
   },
   data() {
-    this.viewAsFilter.filter = this.$route.query.pa || PERMISSIONS.private;
+    this.viewAsFilter.filter =
+      this.$route.query.pa || DISPLAY_LEVELS.private.value;
     return {
       viewAs: this.viewAsFilter.filter,
       viewAsOptions: [
-        { label: 'Myself', value: PERMISSIONS.private, icon: 'avatar' },
-        { label: 'Staff', value: PERMISSIONS.staff, icon: 'staff' },
-        { label: "Ndae'd", value: PERMISSIONS.ndaed, icon: 'triangle' },
-        { label: 'Registered', value: PERMISSIONS.authenticated, icon: 'lock' },
-        { label: 'Public', value: PERMISSIONS.public, icon: 'world' },
+        {
+          label: 'Myself',
+          value: DISPLAY_LEVELS.private.value,
+          icon: 'avatar',
+        },
+        {
+          label: DISPLAY_LEVELS.staff.label,
+          value: DISPLAY_LEVELS.staff.value,
+          icon: DISPLAY_LEVELS.staff.icon,
+        },
+        {
+          label: DISPLAY_LEVELS.ndaed.label,
+          value: DISPLAY_LEVELS.ndaed.value,
+          icon: DISPLAY_LEVELS.ndaed.icon,
+        },
+        {
+          label: DISPLAY_LEVELS.authenticated.label,
+          value: DISPLAY_LEVELS.authenticated.value,
+          icon: DISPLAY_LEVELS.authenticated.icon,
+        },
+        {
+          label: DISPLAY_LEVELS.public.label,
+          value: DISPLAY_LEVELS.public.value,
+          icon: DISPLAY_LEVELS.public.icon,
+        },
```

Comment:
For staff, authenticated, public (and hopefully nda), we can just include the whole `DISPLAY_LEVELS.foobar` object.